### ISSUE TITLE
How to handle the R return matrix value 

### ISSUE THREAD

**Issue by benjamin658** [jakutis/rserve-client#1]
Hi, thanks for your excellent module...
I have one question about the R return matrix value.
For example when I evaluate('matrix(seq(1,25,1), nrow=5, ncol=5, byrow=T)')
I expect that callback will get the value like:

```
    [,1] [,2] [,3] [,4] [,5]
[1,]    1    2    3    4    5
[2,]    6    7    8    9   10
[3,]   11   12   13   14   15
[4,]   16   17   18   19   20
[5,]   21   22   23   24   25
```

but I just got the value without any index, for the example above, I will get the return value like:

```
1 6 11 16 21 2 7 12 17 22 3 8 13 18 23 4 9 14 19 24 5 10 15 20 25
```

Is there any method or clue to handle the matrix value?
Thanks again!

**Comment by jakutis** [comment]
I have just released version 0.3.1, please upgrade and tell if it works for you now.

**Comment by benjamin658** [comment]
yes, I got the return value with index now...
thanks a lot.
